pages/fundamentals/data-types.mdx

```diff
@@ -271,8 +272,8 @@ RETURN actor {.*, .dateOfBirth} as bradley
 
 ## Temporal types
 
-The following temporal types are available: `Duration`, `Date`, `LocalTime` and
-`LocalDateTime`.
+The following temporal types are available: `Duration`, `Date`, `LocalTime`,
+`LocalDateTime` and `ZonedDateTime`.
```

Comment:
Just a comment: When reading this as a user, I would expect to see `DateTime` here because of the consistency issue: Every temporal type (`Duration`, `Date`, `LocalTime` and `LocalDateTime`) match the same keyword in Cypher except for `ZonedDateTime` which should just be `DateTime`.

Reply:
Given this [conversation](https://memgraph.slack.com/archives/C01BRQLQVAA/p1715080603943749?thread_ts=1715072119.857719&cid=C01BRQLQVAA) from Slack, can this be resolved, leaving the docs in their current state?

Reply:
Yep, you can resolve my comment. We will keep it as is.